data/campaigns/Descent_Into_Darkness/scenarios/03_A_Haunting_in_Winter.cfg

```diff
@@ -1380,7 +1380,7 @@ All sense of time and being are entombed within darkness. How long has passed si
 
         [modify_unit]
             [filter]
-                type=Wolf,Great Wolf,Direwolf
+                type_adv_tree=Wolf
```

Comment:
This will not work. The campaign does not use the `{ENABLE_WOLF_ADVANCEMENT}` in its _main.cfg so the wolf does not advance.

Reply:
fixed